Jenkinsfile

```diff
@@ -39,9 +39,33 @@ dockerTemplate {
         }
       }
 
+      stage('Test') {
```

Comment:
nitpick: The commit message fails to mention what issue it is trying to address.

Reply:
@sthaha 'Basic sanity testing for the maven image'. Let's rely less on luck before shipping to prod :)